traveler-swift-core/TravelerKit/Models/CatalogItem.swift

```diff
@@ -9,26 +9,11 @@
 import Foundation
 
 /// An item in the `CatalogGroup`
-public struct CatalogItem: Decodable, Product {
-    /// Identifier
-    public let id: String
+public protocol CatalogItem {
     /// Title
-    public let title: String
+    var title: String { get }
     /// Secondary title
-    public let subTitle: String
+    var subTitle: String? { get }
     /// URL for a thumbnail
-    public let imageURL: URL?
-    /// Price
-    public var price: Price
-    /// Categories
-    public let categories: [Category]
```

Comment:
Do we not need category moving forward?

Reply:
Category now added to the Product protocol